### PR TITLE
added a couple of debug statements working on https://github.com/tess…

### PR DESCRIPTION
**Pull request by Frijol**
…el/t2-firmware/issues/111

r? @johnnyman727

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 //  todo: finish read mem, start anyting related to changing mem contents
 
-var DEBUG = 0; // 1 if debugging, 0 if not
+var DEBUG = 1; // 1 if debugging, 0 if not
```
Comment (johnnyman727) [inline index.js]: We probably want this to be 0 by default.

Comment (Frijol) [inline index.js]: whoops thanks for the catch

**Comment by LinusU** [comment]
A possible future improvement would be to add [debug (the npm module)](https://www.npmjs.com/package/debug) support to all of our modules. It would be super convenient to be able to debug any part of the tessel ecosystem with e.g. `DEBUG=tessel:rfid`.

I've had much success with this module in the past and it's very widely used (15M downloads last month).

**Comment by johnnyman727** [comment]
@LinusU I had the same thought but I'm pretty sure T1 doesn't support `debug` :/ 

But, seeing as we will probably be debugging modules on T2 rather than T1 moving forward, it probably isn't such a bad thing.

**Comment by LinusU** [comment]
Why is that? It's just printing to stderr?

**Comment by Frijol** [comment]
That would be a cool thing to explore – worth an issue on probably t2/firmware?

In the meanwhile, is this good to merge?

**Comment by johnnyman727** [comment]
@LinusU with an error message like this it's anyone's guess:
```
/app/node_modules/debug/node.js:45: attempt to index a nil value
```

@Frijol yes, I think this is good to merge for now.

**Comment by LinusU** [comment]
Hehe, interesting

LGTM :+1:

**Pull request merged**